### PR TITLE
[FIXED JENKINS-52395] Upgrade to groovy-cps 1.29

### PR DESCRIPTION
**Pull request by stuartrowe**
See https://github.com/cloudbees/groovy-cps/pull/98

### CODE REVIEW

**Comment by dwnusbaum** [comment]
Thanks! FWIW feel free to leave this open, once the downstream PR is merged and released you can just update the version here and then we can merge and release.

File: pom.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>3.3</workflow-support-plugin.version>
         <scm-api-plugin.version>2.2.6</scm-api-plugin.version>
-        <groovy-cps.version>1.28</groovy-cps.version>
+        <groovy-cps.version>1.29-20190627.174504-1</groovy-cps.version>
```
Comment (dwnusbaum) [inline pom.xml]: groovy-cps 1.29 was just released.
```suggestion
        <groovy-cps.version>1.29</groovy-cps.version>
```

**Comment by dwnusbaum** [comment]
Looks like a bad rebase/merge?

**Comment by stuartrowe** [comment]
Probably, I don't use git for my day to day work. There's nothing special in my branch so I'll close this PR and recreate it.

File: pom.xml (hunk 1 above)
Comment (dwnusbaum) [reply pom.xml]: FWIW in the future you accept suggestions like this directly from the GUI if desired.

**Pull request closed without merging**